core/templates/dev/head/base.html

```diff
@@ -225,10 +225,44 @@
                             </ul>
                           </li>
                         {% endif %}
+                        {% if nav_mode != 'create' and nav_mode != 'explore' %}
+                          <create-exploration-button ng-if-start="windowWidth > 768"></create-exploration-button>
```

Comment:
Using ng-if-start isn't what I meant -- it's a good find (I was unaware of it before) but it isn't part of the main Angular documentation and I don't want to depend on a feature that's undocumented (and possibly unsupported in the long term).

What I actually meant was to surround the whole thing here with a div that has a single ng-if condition.